pkg/kubectl/cmd/cmd.go

```diff
@@ -171,6 +171,7 @@ __custom_func() {
    * resourcequotas (aka 'quota')
    * replicasets (aka 'rs')
    * replicationcontrollers (aka 'rc')
+   * templates
```

Comment:
This is used by `get` and `describe`, so you'll need to implement its describer too.